common/lifecyleMgr.go

```diff
@@ -137,6 +138,17 @@ func (lcm *lifecycleMgr) watchInputs() {
 		default:
 		}
 
+		allCharsAreWhiteSpace := true
+		for _, ch := range msg {
+			if !unicode.IsSpace(ch) {
```

Comment:
What is this code doing, it waits on a channel until it receives a message with all spaces?
Where do we use this logic.

Reply:
It waits on a channel till it receives a msg with newline at the end. The message is then processed. We changed it to receive special messages (to receive change in throughput etc). But users would hit newlines a bunch of times on their system, and would receive a "Discarding invalid msg" response from AzCopy.